src/07-even-more-loops.js

```diff
@@ -96,7 +142,9 @@ function evenArray(array) {}
  *  findSmallest([ 0, 11, -2, 5 ]);
  *  //> -2
  */
-function findSmallest(array) {}
+function findSmallest(array) {
+  return Math.min(...array);
```

Comment:
Very slick here!   Just make sure we are understand what exactly is happening with our spread operator and Math methods.